.github/workflows/scanner-dev-vuln-update.yaml

```diff
@@ -84,32 +66,45 @@ jobs:
         echo "code: $code"
         echo "$code" | grep -q 200
 
+    - name: Set MANUAL_URL
+      id: set-manual-url
+      run: |
+        if [ "${{ github.event_name }}" == "schedule" ]; then
+          echo "MANUAL_URL=https://raw.githubusercontent.com/stackrox/stackrox/master/scanner/updater/manual/vulns.yaml" >> "$GITHUB_ENV"
+        else
+          echo "MANUAL_URL=https://raw.githubusercontent.com/stackrox/stackrox/${{ github.event.pull_request.head.sha }}/scanner/updater/manual/vulns.yaml" >> "$GITHUB_ENV"
+        fi
+
     - uses: ./.github/actions/upload-artifact-with-retry
       with:
         name: nvd
         path: nvd.zip
         if-no-files-found: error
 
-  run-updater:
+  build-and-upload:
     needs:
-    - build-updater
-    - download-nvd
+    - parse-versions
+    - prepare-environment
     runs-on: ubuntu-latest
-    env:
-      STACKROX_NVD_API_KEY: ${{ secrets.NVD_API_KEY }}
-      STACKROX_NVD_API_CALL_INTERVAL: 6s
-
+    container:
+      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.4.2
+      volumes:
+      # The updater makes heavy use of /tmp files.
+      - /tmp:/tmp
+      - /usr:/mnt/usr
+      - /opt:/mnt/opt
+    strategy:
+      fail-fast: false
+      max-parallel: 1
+      matrix:
+        vb: ${{ fromJson(needs.parse-versions.outputs.versions) }}
```

Comment:
Since this workflow is only for `dev` per PR description, should we remove this matrix and instead build `dev` from `master` (or if multiple dev bundles are needed maintain a separate mapping?).

Reply:
The new dev workflow is changed to new definition update workflow, I also updated the PR description.